postgres-appliance/scripts/configure_spilo.py

```diff
@@ -264,7 +264,7 @@ def deep_update(a, b):
 scope: &scope '{{SCOPE}}'
 restapi:
   listen: ':{{APIPORT}}'
-  connect_address: {{instance_data.ip}}:{{APIPORT}}
+  connect_address: {{REST_API_CONNECT_ADDRESS}}:{{APIPORT}}
```

Comment:
I think it is better to call it `RESTAPI_CONNECT_ADDRESS` so it looks similar to https://patroni.readthedocs.io/en/latest/ENVIRONMENT.html#rest-api

Reply:
OK done. I added also a row in ENVIRONMENT.rst for documentation.
Please check it.